class AuthException(Exception):
    pass

class FileNotFoundException(Exception):
    pass